stages/org.osbuild.bootc.install-to-filesystem

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+"""Run bootc install to-filesystem
+
+Note that this needs the disk.img in the inputs as one continous
+devices so that bootupd can install grub to the mbr.
+
+Buildhost commands used: bootc
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util import containers
+
+CAPABILITIES = ["CAP_MAC_ADMIN"]
+
+SCHEMA_2 = r"""
+"inputs": {
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["images"],
+  "properties": {
+    "images": {
+      "type": "object",
+      "additionalProperties": true
+    }
+  }
+},
+"options": {
+  "additionalProperties": false
+},
+"devices": {
+  "type": "object",
+  "additionalProperties": true
+},
+"mounts": {
+  "type": "array"
+}
+"""
+
+
+def main(inputs, mounts):
+    images = containers.parse_containers_input(inputs)
+    assert len(images) == 1
+    image = list(images.values())[0]
+
+    with containers.container_source(image) as (_, source):
+        env = os.environ.copy()
+        env["BOOTC_SKIP_UNSHARE"] = "true"
```

Comment:
Why out of curiosity?   I did https://github.com/containers/bootc/pull/275 motivated by this, FWIW.

Reply:
@cgwalters Thank you very much for this! The authoritative source for this is @ondrejbudai  - I think we need to do it because we are at this point inside the bubble wrap container in an osbuild buildroot so unshare of a mount namespace will not work. But I'm only 65% confident in this answer and have not tested it myself so take it with a grain of salt.

Reply:
I don't remember the details, sorry. We definitely have `CAP_SYS_ADMIN` in the container, so this isn't an issue. Might be worth trying this without `BOOTC_SKIP_UNSHARE`, but since `bootc` now does this by default, this feels pointless now.

Reply:
Given that this is the default (for now) I will remove setting the env.